moped-database/migrations/1653403352000_insert_milestones/down.sql

```diff
@@ -0,0 +1,40 @@
+-- Reset milestone_order based on milestone_id
+UPDATE public.moped_milestones SET milestone_order = 1 WHERE milestone_id = 1;
+UPDATE public.moped_milestones SET milestone_order = 2 WHERE milestone_id = 2;
+UPDATE public.moped_milestones SET milestone_order = 3 WHERE milestone_id = 3;
+UPDATE public.moped_milestones SET milestone_order = 4 WHERE milestone_id = 4;
```

Comment:
I took a peek at production's database and the seed data, and they both have the same set of "initial" milestones so this is 👍. 

However, would it be easier to do something like `update moped_milestones set milestone_order = milestone_id;`?